LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/IT87XX.cs

```diff
@@ -22,12 +22,17 @@ internal class IT87XX : ISuperIO
         private readonly int _gpioCount;
         private readonly bool _has16BitFanCounter;
         private readonly bool _hasNewerAutoPwm;
-        private readonly byte[] _initialFanPwmControl = new byte[3];
+        private readonly byte[] _initialFanPwmControl = new byte[3]; //this will also store the 2nd control register value
```

Comment:
Add a space and capitalize the first letter of each comment you've added, in order to follow code style. Also add a dot at the end.